backend/de.metas.business/src/main/java/de/metas/quantity/Quantitys.java

```diff
@@ -89,7 +89,7 @@ public Quantity createZero(@NonNull final UomId uomId)
 		return Quantity.zero(uomRecord);
 	}
 
-	public Quantity one(@NonNull final UomId uomId)
+	public Quantity createOne(@NonNull final UomId uomId)
```

Comment:
TBH, I would prefer "one". Feels much more concise, "create" is obvious, modern.
Instead I would rename the other methods, .e.g.
* createZero(@NonNull final UomId uomId) -> `zero`
* create(@NonNull final BigDecimal qty, @NonNull final X12DE355 x12DE355) -> `of`